text/0202-events-and-logs-api.md

```diff
@@ -0,0 +1,172 @@
+
+### Logger
+
+The Logger is responsible for creating Events and Logs.
+
+Note that Loggers should not be responsible for configuration. This should be the responsibility of the LoggerProvider instead.
+
+#### Logger operations
+
+The Logger MUST provide functions to:
+
+- Create an `Event` and emit it to the processing pipeline.
+  - The API MUST accept an event name as a parameter. The event name provided should be inserted as an attribute with key `event.name`. Care MUST be taken to not override or delete this attribute while the `Event` is created.  This function MAY be named `logEvent`.
+- Create a `Log Record` and emit it to the processing pipeline.
+  - This function MAY be named `logRecord`.
+  - The intended users of this API is Log Appenders.
+
+### Usage
+
+```java
+OpenTelemetry openTelemetry = OpenTelemetry.noop();
+Logger logger = openTelemetry.getLogger("my-scope");
+
+// Using the convenience method to log an event directly
+logger.logEvent("network-changed", 
+                 Attributes.builder().put("type", "wifi").build());
+
+// Using the event builder to log an event
+logger.eventBuilder("page-navigated").build().setAttribute("url", "http://foo.com/bar#new").emit();
+
+// Using the logRecord builder to log a record
+Logger logger = openTelemetry.getLogger("another-scope");
+logger.logRecordBuilder().build().setBody("I am a log message").emit();
+
+```
+
+### Usage in Client-side telemetry
+
+```java
+public void addBrowserEvent(String name, Attributes attributes) {
+   Logger logger = openTelemetry.getLogger("my-scope", "1.0", "browser");
+   logger.logEvent(name, attributes);
+}
+
+public void addMobileEvent(String name, Attributes attributes) {
+   Logger logger = openTelemetry.getLogger("my-scope", "1.0", "mobile");
+   logger.logEvent(name, attributes);
+}
+```
+
+### Usage in eBPF
+
+From the eBPF [demo](https://youtu.be/F1VTRqEC8Ng?t=233), it looks like they have chosen to put the Event data in the Body field of LogRecord instead of Attributes.  They might have to make the changes to move the event data to attributes to conform to this API specification.
+
+## Semantic Convention for event attributes
+
+**type:** `event`
+
+**Description:** Event attributes.
+
+| Attribute  | Type | Description  | Examples  | Required |
+|---|---|---|---|---|
+| `event.name` | string | Name or type of the event. | `network-change`; `button-click`; `exception` | Yes |
+| `event.domain` | string | Domain or scope for the event. | `profiling`; `browser`, `db`, `k8s` | No |
+
+An `event.name` is supposed to be unique only in the context of an `event.domain`, so this allows for two events in different domains to have same `event.name`. No claim is made about the uniqueness of `event.name`s in the absence of `event.domain`.
+
+Note that Scope attributes are equivalent to the attributes at Span and LogRecord level, so recording the attribute `event.domain` on the Scope is equivalent to recording it on Spans and LogRecords within the Scope.
```

Comment:
I am not against this, but we need to clarify this behavior generally in the spec, not just for logs but also for traces and metrics.